### PR TITLE
Release notes--please review :)

### PR DESCRIPTION
**Pull request by ruchim**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,14 @@ parameters as a HTTP POST.
 
 ## 0.21
 
+
```
Comment (kcibul) [inline CHANGELOG.md]: Can you add something like

+* Cromwell has an entirely new internal architecture which scales much better, and will allow for cleaner future extension

**Comment by geoffjentry** [comment]
I think it'd be good to right at the top make a big point about the number one 'feature' here is enhanced performance and scalability

File: CHANGELOG.md [hunk 2]
```diff
@@ -21,6 +21,8 @@ parameters as a HTTP POST.
 * Warning: Significant database updates when you switch from version 0.19 to 0.21 of Cromwell.
   Please be refer to MIGRATION.md for more details.
 
+* There are significant architectural changes related related to increases in performance and scaling.
```
Comment (Horneth) [inline CHANGELOG.md]: double `related`

File: CHANGELOG.md [hunk 3]
```diff
@@ -17,11 +17,54 @@ parameters as a HTTP POST.
 
 ## 0.21
 
+
+* Warning: Significant database updates when you switch from version 0.19 to 0.21 of Cromwell.
+  Please be refer to MIGRATION.md for more details.
```
Comment (mcovarr) [inline CHANGELOG.md]: Please be correcting "Please be refer"

File: CHANGELOG.md [hunk 4]
```diff
@@ -17,11 +17,54 @@ parameters as a HTTP POST.
 
 ## 0.21
 
+
+* Warning: Significant database updates when you switch from version 0.19 to 0.21 of Cromwell.
```
Comment (mcovarr) [inline CHANGELOG.md]: I'd suggest rewording this to say that the database migration will run for a long time if there are large amounts of data already in the database.

File: CHANGELOG.md [hunk 5]
```diff
@@ -17,11 +17,54 @@ parameters as a HTTP POST.
...
+
+* The biggest changes from 0.19 to 0.21 are related to the application.conf file, which has been restructured significantly.
+The configuration for backends now is all contained within a `backend` stanza, which specifies 1 stanza per name per backend and a default backend, as follows:
+
+```
+backend {
+    default=Local
+    providers {
+        Local {
+            actor-factory: "class path to BackendLifecycleActorFactory implementation"
```
Comment (mcovarr) [inline CHANGELOG.md]: these are the fully qualified names of the actor factory classes

File: CHANGELOG.md [hunk 6]
```diff
@@ -17,11 +17,54 @@ parameters as a HTTP POST.
 
 ## 0.21
 
+
+* Warning: Significant database updates when you switch from version 0.19 to 0.21 of Cromwell.
+  Please be refer to MIGRATION.md for more details.
+
+* There are significant architectural changes related related to increases in performance and scaling.
+
+* The biggest changes from 0.19 to 0.21 are related to the application.conf file, which has been restructured significantly.
```
Comment (mcovarr) [inline CHANGELOG.md]: the biggest user-facing changes, perhaps 😄

File: CHANGELOG.md (hunk 4 above)
Comment (Horneth) [inline CHANGELOG.md]: This is said in the migration.md as well but doesn't hurt to say it twice xD
